core/src/consensus/fork_choice.rs

```diff
@@ -22,6 +22,16 @@ pub struct SelectVoteAndResetForkResult {
     pub heaviest_fork_failures: Vec<HeaviestForkFailures>,
 }
 
+struct CandidateVoteAndResetBanks<'a> {
```

Comment:
Pulled this struct definition outside of the `select_vote_and_reset_forks` function